.github/workflows/ci.yaml

```diff
@@ -13,14 +13,19 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
+    env:
+      SYMFONY_REQUIRE: ${{ matrix.symfony-require }}
```

Comment:
is it somehow used by composer to require symfony of particular version during `composer update`? As I don't see where we explicitly require the needed version (`^5`, `^6` ...)

Reply:
symfony flex does it for us.
https://github.com/symfony/flex/blob/7f45cf796f591ad8c1cc56ccc8edc32d78172258/src/Flex.php#L125